### PR TITLE
Fix basic requirements of `proof`

### PR DESCRIPTION
**Pull request by TallTed**
***
<a href="https://pr-preview.s3.amazonaws.com/TallTed/vc-data-integrity/pull/166.html" title="Last updated on Aug 14, 2023, 7:41 PM UTC (70813d2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/166/6c94d0b...TallTed:70813d2.html" title="Last updated on Aug 14, 2023, 7:41 PM UTC (70813d2)">Diff</a>

### CODE REVIEW

**Review by Wind4Greg — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -532,9 +532,9 @@ <h3>Proofs</h3>
 
         <p>
 When expressing a <a>data integrity proof</a> on an object, a
-<dfn class="lint-ignore">`proof`</dfn> property MUST be used where its value is a
-single object, or an unordered set of objects, expressed using the properties
-below:
+<dfn class="lint-ignore">`proof`</dfn> property MUST be used. If present, its
+value MUST be either a single object, or an unordered set of objects, expressed
```
Comment (msporny) [inline index.html:536]:
```suggestion
<dfn class="lint-ignore">`proof`</dfn> property MUST be used and its
value MUST be either a single object, or an unordered set of objects, expressed
```

It's a bit strange to say "If present" for a property that MUST be present? Minor rewording suggestions.

**Review by dlongley — APPROVED** [review]
Approving w/@msporny's suggestion.

File: index.html (hunk 1 above)
Comment (TallTed) [reply index.html:536]: @msporny @dlongley -- Reading the whole paragraph, it's only a MUST "When expressing a <a>data integrity proof</a> on an object", so "if present" seems OK.

I think the "If present" phrasing covers any time the `proof` property is present, while the "and" phrasing only covers the `proof` property "When expressing a <a>data integrity proof</a> on an object".

Some additional rephrasing, possibly including other segments of the doc, may be needed to make the restrictions on the `proof` property clearer.

Comment (msporny) [reply index.html:536]: If you feel strongly about this, I'm fine w/ leaving it as is. I don't think the new wording I suggested changes the meaning of the sentence.

Comment (TallTed) [reply index.html:536]: I prefer as-is, because this doesn't spark the need for further review/rewording.

**Review by msporny — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Pull request merged**